Automattic/woocommerce-payments
# Fix Apple Pay and Google Pay if card payments are disabled

Fixes #5945 

#### Changes proposed in this Pull Request


If card payment method is disabled, the payments with Apple Pay and Google Pay fails. We need to remove the validation for PM, to allow creating a Payment Intent without a payment method.






#### Testing instructions


* checkout WcPay 5.3.0 or older.
* Enable UPE.
* Disable the Debit / Credit card option.
* Swith to this branch
* The Debit / Credit card option cannot be enabled.
* Attempt to make a purchase via Google Pay or Apple Pay.
* The order should be successful.


*

-------------------


**Post merge**
